torchquad/plots/plot_adaptive_grid.py

```diff
@@ -0,0 +1,29 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def plot_grid(grid, dpi=100):
+    """Plots the adaptive grid and corresponding function value.
+
+    Args:
+        grid (AdaptiveGrid): AdaptiveGrid of evaluated function
+        dpi (int, optional): Plot dpi. Defaults to 100.
+    """
+
+    fig = plt.figure(dpi=dpi)
+    points = None
+    fvals = None
+    for subdomain in grid.subdomains:
+        if points is None:
+            points = subdomain.points
+            fvals = subdomain.fval
+        else:    
+            points = np.concatenate([points,subdomain.points])
+            fvals = np.concatenate([fvals,subdomain.fval])
```

Comment:
```suggestion
        if points is None:
            points = subdomain.points.cpu().numpy()
            fvals = subdomain.fval.cpu().numpy()
        else:    
            points = np.concatenate([points,subdomain.points.cpu().numpy()])
            fvals = np.concatenate([fvals,subdomain.fval.cpu().numpy()])
```

Otherwise you get a
`TypeError: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.` if you use the GPU